blocks/utils/index.js

```diff
@@ -62,3 +62,23 @@ export function isLegacyForm(forms, SelectedFormId) {
 
```

Comment:
I'm thinking about do we really need this file with these helper functions. We could probably move everything into `DonationFormBlock` `Edit.tsx`

Reply:
Resolved: https://github.com/impress-org/givewp/pull/7082/commits/445155c66d0f359fe15b65683a1b0a81b731b8b6